src/api.cpp

```diff
@@ -904,6 +905,18 @@ cl_int CLVK_API_CALL clGetDeviceInfo(cl_device_id dev,
         copy_ptr = &val_pci_bus_info;
         size_ret = sizeof(val_pci_bus_info);
         break;
+    case CL_DEVICE_SUB_GROUP_SIZES_INTEL:
+        if (device->supports_controled_subgroups()) {
+            uint32_t size = device->min_sub_group_size();
+            while (size <= device->max_sub_group_size()) {
+                subgroup_sizes.push_back((size_t)size);
+                size *= 2;
+            }
```

Comment:
Are all sizes in between guaranteed to be supported?

Reply:
That's what I understand from the specification, but it is not stated as such.